src/pages/products/testing-center/index.tsx

```diff
@@ -0,0 +1,168 @@
+          <div className={styles.grid}>
+            <ValuePropCard title="The most secure exams">
+              Maximize test integrity with managed computers, lab firewall, and
+              vigilant proctoring. Students access only approved resources,
+              while cheating risks are minimized through computer screens, video
+              cameras and professional oversight.
+            </ValuePropCard>
+            <ValuePropCard title="Reliable check-in process">
+              Enhance security measures by verifying students&apos; identities
+              at the lab entrance through their student ID cards. In addition,
+              all personal belongings, including phones, are securely stored
+              before entering the testing environment.
+            </ValuePropCard>
+            <ValuePropCard title="Flexible and efficient scheduling">
+              Offer exams asynchronously over 2-3 days, optimizing space
+              utilization in smaller computer labs for larger classes. Eliminate
+              the need for instructors to manage make-up or conflict exams,
+              streamlining the testing process for both students and faculty.
+            </ValuePropCard>
+            <ValuePropCard title="Student-centric exam scheduling">
+              Empower students with the flexibility to schedule exams at their
+              preferred times within the exam period. Enable easy reservation
+              modifications and cancellations up to the selected exam time,
+              providing convenience and autonomy to students.
+            </ValuePropCard>
+            <ValuePropCard title="Seamless exam overrides">
+              Empower instructors with the capability to effortlessly grant
+              overrides for students facing unexpected circumstances, such as
+              illness or unavoidable absences during the exam period.
+            </ValuePropCard>
+            <ValuePropCard title="Authentic testing">
+              Create an authentic testing environment by providing students with
+              access to computers equipped with compilers, debuggers, and
+              graphical software, ensuring a comprehensive assessment of their
+              skills and knowledge.
+            </ValuePropCard>
+          </div>
+        </div>
+      </div>
+
+      <div className={classnames("container-fluid py-4", styles.container)}>
+        <div className="container-md">
+          <div className="row">
+            <div className="col-md-6 order-1 pt-4">
+              <Image
+                src={reducedDistractionImage}
+                alt="seat in reduced distraction environment"
+                style={{
+                  maxWidth: "100%",
+                  height: "auto",
+                }}
+              />
+            </div>
+            <div className="col-md-6 order-2 pt-4">
+              <div className="row">
+                <Heading>Academic Accommodations</Heading>
```

Comment:
Here and elsewhere I think we should switch to sentence case for headings. This is for consistency with the main PL page and I think it's better anyway.

Reply:
resolved